### PR TITLE
[T2A4][F12-A2]Lee Jun Han Bryan

### PR DESCRIPTION
**Pull request by bryanleejh**
Ready for review.

Will look at sort feature again soon.

### CODE REVIEW

**Review by brandonyeoxg — COMMENTED** [review]
Made edits below.

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -1126,14 +1153,20 @@ private static String getUsageInfoForViewCommand() {
...
-        return String.format(MESSAGE_COMMAND_HELP, COMMAND_EXIT_WORD, COMMAND_EXIT_DESC)
-                + String.format(MESSAGE_COMMAND_HELP_EXAMPLE, COMMAND_EXIT_EXAMPLE);
+        return String.format(MESSAGE_COMMAND_HELP, COMMAND_EXIT_WORD, COMMAND_EXIT_DESC) + LS
+                + String.format(MESSAGE_COMMAND_HELP_EXAMPLE, COMMAND_EXIT_EXAMPLE) + LS;
+    }
+    
+    /** Returns the string for showing 'sort' command usage instruction */
+    private static String getUsageInfoForSortCommand() {
+    	return String.format(MESSAGE_COMMAND_HELP, COMMAND_SORT_WORD, COMMAND_SORT_DESC) + LS
+    			+ String.format(MESSAGE_COMMAND_HELP_EXAMPLE, COMMAND_SORT_EXAMPLE) + LS;
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/AddressBook.java:1169]: Do note again that in line 1169, there is tab instead of space.

**Review by brandonyeoxg — COMMENTED** [review]
@bryanleejh Some comments added. Please close the PR after reading comments. Glad to see that you have done testing! Keep it up.

**Comment by bryanleejh** [comment]
Thanks (:

**Pull request closed without merging**